index.html

```diff
@@ -38,6 +38,107 @@ <h2 class="connect">LET'S CONNECT</h2>
             </ul>
         </section>
 
+        <section class="portfolio">
+            <div class="project-cards">
+                <div class="cards card1">
+                    <img src="file:///C:/All%20stuff/Microverse/My-Portfolio/images/Snapshoot%20Portfolio.svg" alt="Snapshot Portfolio">
+                    <h3 class="project-name">Tonic</h3>
+                        <div class="key-text"> 
+                            <p class="canopy">CANOPY</p>
+                            <p class="fa-solid fa-circle fa-2xs"></p>
+                            <p class="position">Back end dev</p>
+                            <p class="fa-solid fa-circle fa-2xs"></p>
+                            <p class="year">2015</p>
+                        </div>
+                        <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <ul class="language-container">
+                            <li class="lang">html</li>
+                            <li class="lang">css</li>
+                            <li class="lang">javascript</li>
+                        </ul>
+                        <button class="see-project">See Project</button>
```

Comment:
Kindly fix the spacing b/n the `li` to match the Figma design. You can replace the `justify-content: space-around;` by `gap: 10px`. Also kindly fix the background color for the `See Project` button to match the Figma design.

| Your Design | Figma Design|
|--------|--------|
| <img width="302" alt="image" src="https://user-images.githubusercontent.com/34578631/159096788-15051ab0-4272-4847-9cea-a0a818149aca.png"> | <img width="282" alt="image" src="https://user-images.githubusercontent.com/34578631/159097013-77d08f52-6c87-42c4-a496-e93529aba032.png"> |